### DISCUSSION TITLE
Welcome to Kingroon_KP5L_Marlin_Firmware Discussions!

### DISCUSSION THREAD

**Discussion by mechano** [mechano/Kingroon_KP5L_Marlin_Firmware#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UltraLordEdition** [comment]
Hi to all!
I'm developing new print head for KP5L. The page of project -> [LPH for Kingroon KP5L](https://github.com/UltraLordEdition/LPH_for_Kingroon_KP5L)

**Comment by MrMores** [comment]
Hey!
I require a little assistance; I installed your manual leveling firmware and leveled the bed, I then added the M420 S1 command to my g-code like you suggested. I also have a 3d touch on my printer and at the time it was connected to the Z+ pins. And so when I started a print my printer wouldn't even start the purge process after heating up the extruder. Any ideas?

Also here is my starting g-code (I am very new to this): 
G28 ; home all axes
 M420 S1
 M117 Purge extruder
 G92 E0 ; reset extruder
 G1 Z1.0 F3000 ; move z up little to prevent scratching of surface
 G1 X2 Y20 Z0.3 F5000.0 ; move to start-line position
 G1 X2 Y60 Z0.3 F1500.0 E15 ; draw 1st line
 G1 X2 Y60 Z0.4 F5000.0 ; move to side a little
 G1 X2 Y20 Z0.4 F1500.0 E30 ; draw 2nd line
 G92 E0 ; reset extruder
 G1 Z1.0 F3000 ; move z up little to prevent scratching of surface
